### PR TITLE
UI improvement to blog index page

### PR DESCRIPTION
**Pull request by devkiran**

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -134,8 +134,8 @@ module.exports = {
             type: 'all',
             copyright,
           },
-          blogSidebarTitle: 'All posts',
-          blogSidebarCount: 'ALL',
+          blogSidebarTitle: 'Recent posts',
+          blogSidebarCount: 5,
```
Comment (deepakprabhakara) [inline docusaurus.config.js:138]: Any reason to not show all blogs on the sidebar?

**Comment by niwsa** [comment]
Just thinking out loud. The card looks good. Is the author details required in the card. What if we show the tags associated with the blog instead ?

**Comment by devkiran** [comment]
@niwsa I'm not sure. I just kept all the details we have available for a post item. Since the amount of blog is very less (just 10 now), the tag doesn't make sense. What do you think?

File: docusaurus.config.js (hunk 1 above)
Comment (devkiran) [reply docusaurus.config.js:138]: No particular reason. I thought about showing only the latest 5 to reduce the list items in the sidebar.

**Comment by deepakprabhakara** [comment]
Can we have h1 as Blog and h2 a short subtitle (will post the exact message soon)

**Comment by devkiran** [comment]
> Can we have h1 as `Blog` and h2 a short subtitle (will post the exact message soon)

Yes, we can. Please share the message for h2.

**Review by deepakprabhakara — APPROVED** [review]

**Pull request merged**